tests/tools/traffic-replay/h2Replay.py

```diff
@@ -215,58 +219,61 @@ def txn_replay(session_filename, txn, proxy, result_queue, h2conn,request_IDs):
                 del txn_req_headers_dict['Content-Length']
                 mbody = gen()
         if 'Content-Length' in txn_req_headers_dict:
-            nBytes=int(txn_req_headers_dict['Content-Length'])
+            nBytes = int(txn_req_headers_dict['Content-Length'])
             mbody = createDummyBodywithLength(nBytes)
         if 'Connection' in txn_req_headers_dict:
             del txn_req_headers_dict['Connection']
         #str2 = extractHeader.extract_host(txn_req_headers)+ extractHeader.extract_GET_path(txn_req_headers)
         #print(str2)
         if method == 'GET':
-            responseID = h2conn.request('GET',url=extractHeader.extract_GET_path(txn_req_headers),headers=txn_req_headers_dict, body=mbody)
+            responseID = h2conn.request(
+                'GET', url=extractHeader.extract_GET_path(txn_req_headers), headers=txn_req_headers_dict, body=mbody)
             #print("get response", responseID)
             return responseID
             #request_IDs.append(responseID)
             #response = h2conn.get_response(id)
             #print(response.headers)
             #if 'Content-Length' in response.headers:
             #        content = response.read()
-                    #print("len: {0} received {1}".format(response.headers['Content-Length'],content))
+            #        print("len: {0} received {1}".format(response.headers['Content-Length'],content))
```

Comment:
@dragon512 Actually this space is added by hand, because I saw a error after ran yapf. It changed following line ( `elif method == 'POST':` ) to something like below.
```
method == 'POST':
elif 
```

After adding space, yapf working well.